docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -92,63 +95,168 @@ You can also build the collector-contrib repository by cloning it and running:
 make otelcontribcol
 ----
 
-[float]
-[[open-telemetry-elastic-configure]]
-==== Configure the collector
+[[open-telemetry-elastic-traces-metrics]]
+==== Collect traces and metrics
 
-Create a `yaml` configuration file.
+NOTE: This guide assumes your services and applications have already been instrumented with the OpenTelemetry API and/or SDK.
+If you are new to APM, we recommend <<install-and-run,getting started with Elastic APM agents>> instead.
 
-At a minimum, you must define the URL of the APM Server instance you are sending data to.
-For example:
+To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications
+with the OpenTelemetry API and/or SDK.
 
-[source,yml]
+Here is an example of how to set up the OpenTelemetry Java agent.
+
+[source,bash]
 ----
-exporters:
-  elastic:
-    apm_server_url: "https://elasticapm.example.com"
+export OTEL_RESOURCE_ATTRIBUTES=service.name=frontend,service.version=1.0-SNAPSHOT,deployment.environment=staging
+java -javaagent:/path/to/opentelemetry-javaagent-all-0.10.1.jar \
+     -Dotel.otlp.endpoint=my-otel-collector.mycompany.com:55680 \
+     -jar target/frontend-1.0-SNAPSHOT.jar
 ----
 
-See the <<open-telemetry-elastic-config-ref,configuration reference>> for additional configuration options,
-like specifying an API key, secret token, or TLS settings.
-
-The Elastic exporter must also be defined in `service.pipelines.traces.exporters`.
-For example:
+Here is an example of how to capture business metrics from an application.
 
-[source,yml]
+[source,java]
 ----
-service:
-  pipelines:
-    traces:
-      exporters: [elastic]
+// initialize metric
+Meter meter = OpenTelemetry.getGlobalMeter("my-frontend");
+DoubleCounter orderValueCounter = meter.doubleCounterBuilder("order_value").build();
+
+public void createOrder(HttpServletRequest request) {
+
+   // create order in the database
+   ...
+   // increment business metrics for monitoring
+   orderValueCounter.add(orderPrice);
+}
 ----
 
-If we put everything together, here's an example configuration file that accepts input from an OpenTelemetry Agent,
-processes the data, and sends it to an {ess} instance.
+For more information on setting up OpenTelemetry, see the OpenTelemetry's {ot-collector}[Getting Started]
+documentation.
```

Comment:
I would change this to either this:
```diff
- For more information on setting up OpenTelemetry, see the OpenTelemetry's {ot-collector}[Getting Started]
+ For more information on setting up OpenTelemetry, see the OpenTelemetry {ot-collector}[Getting Started]
```

Or this:
```diff
- For more information on setting up OpenTelemetry, see the OpenTelemetry's {ot-collector}[Getting Started]
+ For more information on setting up OpenTelemetry, see OpenTelemetry's {ot-collector}[Getting Started]
```
For more information on setting up OpenTelemetry, see the OpenTelemetry's {ot-collector}[Getting Started]
documentation.